src/QVClient.ts

```diff
@@ -157,6 +153,14 @@ export interface IQVModel extends IQVModelEdit {
   credentialMetadata?: ICredentialMetadata;
 }
 
+export enum Mode {
+  Begin,
```

Comment:
Add one more for "Not Yet Opened".

Reply:
Should `NotYetOpened` be returned when `modes.stage` is equal to `-1`? 

If that's the case, isn't that the same as `Mode.Begin`? Should I remove `Mode.Begin` and replace it with `Mode. NotYetOpened`?